mission_yaml/SEA45_M48.yml

```diff
@@ -317,37 +316,37 @@ netcdf_variables:
     units:        m
     coordinates:  time depth latitude longitude
 
-  ad2cp_beam1_cell_number1:
-    source:       AD2CP_V1_CN1
+  ad2cp_beam1_cell_number_1:
+    source:       AD2CP_V1_CN[1]
     long_name:    glider beam 1 measure from AD2CP in cell 1
-    standard_name: platform_beam1_measure_ad2cp_cell1
+    standard_name: platform_beam1_measure_ad2cp_cell_1
     units:        m s-1
     coordinates:  time depth latitude longitude
```

Comment:
These `source` lines should match the headers in the pld1 files. I've just checked and these look like AD2CP_V1_CN1 AD2CP_V2_CN1 etc. I don't know where these [] brackets snuck in, bu they should not be there